aces/Loris
# [Help Editor] Reactifying module

This pull request `reactifies the help editor module`. The table should now automatically filter out rows based on the `Help Topic` and `Search Keyword`.

To test:
1. Navigate to the Help Editor.
2. Search for various words and see if the table is populated with correct rows.
3. Try using a search combination of both the `Help Topic` and `Search Keyword` fields.

Worked on with: @colezweber @jesscall @miebeers
